packages/snaps-controllers/src/snaps/SnapController.test.tsx

```diff
@@ -615,7 +616,17 @@ describe('SnapController', () => {
       getSnapControllerOptions({
         messenger,
         state: {
-          snaps: getPersistedSnapsState(),
+          snaps: getPersistedSnapsState(
+            getPersistedSnapObject({
```

Comment:
We should have a test for updating when the installed Snap has both 0 connections as well as some existing connections

Reply:
Added one more test with 0 pre-existing connections. Should be fair enough I guess? Coverage issues are resolved as well.